src/treeview/writer.js

```diff
@@ -414,10 +342,19 @@ import DocumentFragment from './documentfragment.js';
 	 * and {@link engine.treeView.Range#end} positions are not placed inside same parent container.
 	 *
 	 * @param {engine.treeView.Range} range Range to wrap.
-	 * @param {engine.treeView.Element} attribute Attribute element to use as wrapper.
+	 * @param {engine.treeView.AttributeElement} attribute Attribute element to use as wrapper.
 	 * @param {Number} priority Priority to set.
 	 */
 	wrap( range, attribute, priority ) {
```

Comment:
`AttributeElement` store priority itself so there is no need now to pass it as third parameter.